From: Thorsten Glaser <tg@mirbsd.org>
Date: Sat, 28 Jan 2012 16:14:18 +0000
Subject: fix compile time warning on armel, armhf, ia64

utimes is defined in <sys/time.h> only if Fefe’s variant
if _BSD_SOURCE is set, which we don’t do in dietlibc it‐
self, so pull in the declaration

--- a/lib/__utime.c
+++ b/lib/__utime.c
@@ -4,6 +4,8 @@
 #include <sys/time.h>
 
 #ifndef __NR_utime
+extern int utimes(const char *, struct timeval *);
+
 int utime(const char *filename, const struct utimbuf *times)
 {
   if (times == NULL)
